core/src/main/java/com/scalar/db/api/Admin.java

```diff
@@ -402,4 +402,15 @@ void repairTable(
    */
   void addNewColumnToTable(String namespace, String table, String columnName, DataType columnType)
       throws ExecutionException;
+
+  /**
+   * Import an existing table that is not managed by ScalarDB.
+   *
+   * @param namespace an existing namespace
+   * @param table an existing table
+   * @throws IllegalArgumentException if the table is already managed by ScalarDB, if the target
+   *     table does not exist, or if the table does not meet the requirement of ScalarDB table
+   * @throws ExecutionException if the operation fails
+   */
+  void importTable(String namespace, String table) throws ExecutionException;
```

Comment:
That's a good question. Behind this question, are you wondering if we should only support transaction API for importing? It's a decision for what use cases should be covered. I originally thought covering both use cases (storage and transaction) would be nice. But, now I'm not really sure since the major reason for migrating to ScalarDB would be that users want to use transactions in ScalarDB. I would like to also hear other reviewers' thoughts.

Reply:
@feeblefakie We are discussing whether we should support table importing in both storage and transaction API. What do you think?

Reply:
@jnmt I think supporting only transactions seems fine to me.
Can we add support for storage if we really want to do it later on?

Reply:
@jnmt I discuss it with Toshi, and we think we can keep it as is.
Later, we might remove the support for storage, though.

Reply:
@feeblefakie @Torch3333 Got it! Let's keep it as is. Thank you! (I was just wondering about the revert cost of the current PR.)